charts/actions-runner-controller/templates/actionsmetrics.role.yaml

```
@@ -0,0 +1,90 @@
+{{- if .Values.githubWebhookServer.enabled }}
```

```suggestion
{{- if .Values.actionsMetricsServer.enabled }}
```

Shouldn't this refer to the actionsMetricsServer.enabled as it's for actions-metrics-server, not for github-webhook-server?

---

Of course you're right, my bad.